cmake.erfa.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=@CMAKE_INSTALL_PREFIX@
+libdir=@pc_libdir@
+includedir=@pc_includedir@
+
+Name: Erfa
+Description: Essential Routines for Fundamental Astronomy
+Version: @ERFA_VERSION@
```

Comment:
Why do you need another .pc file?

Reply:
Good point, looks like it's possible to use the existing one (unlike `config.h.in`, where there are some formatting differences). Updated.